docs/launch-scenarios.md

```diff
@@ -43,14 +55,42 @@ The app then [subscribes](/#app-subscribes-to-session) to the identified session
 
 Two different launch scenarios are supported. For each launch scenario, the app discovers the session topic to which it [subscribes](/#app-subscribes-to-session).
 
-## EHR Launch: User SSO's into app from EHR
 
-The simplest launch scenario is the [SMART on FHIR EHR launch](http://www.hl7.org/fhir/smart-app-launch/#ehr-launch-sequence), in which the app is launched from an authenticated EHR session. The app receives information about the user and session as part of the launch and subsequently subscribes to the launching user's session. 
+### EHR Launch: User SSO's into app from EHR
+
+The simplest launch scenario is the [SMART on FHIR EHR launch](http://www.hl7.org/fhir/smart-app-launch/#ehr-launch-sequence), in which the subscribing app is launched from an EHR authenticated session. The app receives information about the user and session as part of the launch and subsequently subscribes to the launching user's session. 
 
-In this scenario, the EHR authorizes the app to synchronize. The EHR provides one or more session topics as SMART launch parameters that belong to the EHR's current user. 
+In this scenario, the EHR authorizes the app to synchronize. The EHR provides a session topic as a SMART launch parameter which belongs to the EHR's current user. 
 
-## Standalone launch: User authenticates to EHR to authorize synchronization
+### Standalone launch: User authenticates to EHR to authorize synchronization
 
 If the app can't be launched from the EHR, for example, it's opening on a different machine, it can use the standard [SMART on FHIR standalone launch](http://www.hl7.org/fhir/smart-app-launch/#standalone-launch-sequence). 
 
-In this scenario, the user authorizes the app to synchronize to her session by authenticating to the EHR's authorization server. The EHR provides one or more session topics as SMART launch parameters that belong to the authorizing user.
+In this scenario, the user authorizes the app to synchronize to her session by authenticating to the EHR's authorization server. The EHR provides a session topic as a SMART launch parameter which belongs to the EHR's authorizing user. 
```

Comment:
In SMART, a regular launch only requires the "launch" scope to get context returned. A standalone launch also requires special launch/patient (or encounter) scopes so that the auth server knows those need to be established. For FHIRCast, is the presence of the fhircast scope (in both scenarios) what the auth server would trigger off of?

Reply:
Talked with Jenni, should also clarify that in a standalone launch that `launch/patient` is not used with a `fhircast` scope.

Reply:
Jenni, I updated this language a bit to address. Great comment!